database/migrations/2022_11_08_181911_change_type_user_sign_langague_translations.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Not really a functional issue but can you fix the typo in this filename?

Reply:
Good catch. I've pushed up a fix.